setup.py

```diff
@@ -50,29 +46,18 @@
 LooseVersion(version)  # check that it's a valid version
 
 # Cython extensions
-ext_c_paths = ('eelbrain/*%s', 'eelbrain/_trf/*%s', 'eelbrain/_stats/*%s')  # C
-ext_cpp_paths = ('eelbrain/_stats/*%s',)                                    # C++
```

Comment:
This would add files for `eelbrain/_stats/*.pyx` twice, which seemed like it was asking for trouble...

Reply:
I see!